tests/test_analysis.py

```diff
@@ -1,19 +1,27 @@
 """Analysis module tests"""
 import pytest
-from brevia.analysis import (
+from brevia.tasks.text_analysis import SummarizeTextAnalysisTask
```

Comment:
can we move this testcases to `tests/tasks/test_text_analysys.py` since `brevia/analysis.py` was removed?